torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1093,13 +1093,15 @@ XLATensor XLATensor::div(const XLATensor& input, const XLATensor& other,
   ir::Value other_value = GetFloatingIrValue(other, scalar_type);
   ir::Value res = input_value / other_value;
 
-  if (rounding_mode == "trunc") {
-    res = ir::ops::Trunc(res);
-  } else if (rounding_mode == "floor") {
-    res = ir::ops::Floor(res);
-  } else if (rounding_mode != "true") {
-    XLA_CHECK(false)
-        << "rounding_mode must be one of 'true', 'trunc', or 'floor'";
+  if (rounding_mode.has_value()) {
+    if (*rounding_mode == "trunc") {
+        res = ir::ops::Trunc(res);
+    } else if (*rounding_mode == "floor") {
+        res = ir::ops::Floor(res);
+    } else {
+        XLA_CHECK(false)
+            << "rounding_mode must be one of None, 'trunc', or 'floor'";
+    }
```

Comment:
I think we should also update the `rounding_mode != "true"` check below to `rounding_mode.has_value()`